src/start/cp2k_runs.F

```diff
@@ -7,133 +7,137 @@
-        section_vals_release, section_vals_retain, section_vals_type, section_vals_val_get, &
-        section_vals_write, write_section_xml
-   USE ipi_driver,                      ONLY: run_driver
-   USE kinds,                           ONLY: default_path_length,&
-                                              default_string_length,&
-                                              dp,&
-                                              int_8
-   USE library_tests,                   ONLY: lib_test
-   USE machine,                         ONLY: default_output_unit,&
-                                              m_chdir,&
-                                              m_flush,&
-                                              m_getcwd,&
-                                              m_memory,&
-                                              m_memory_max,&
-                                              m_walltime
-   USE mc_run,                          ONLY: do_mon_car
-   USE md_run,                          ONLY: qs_mol_dyn
-   USE message_passing,                 ONLY: mp_any_source,&
-                                              mp_comm_type,&
-                                              mp_para_env_release,&
-                                              mp_para_env_type
-   USE mscfg_methods,                   ONLY: do_mol_loop,&
-                                              loop_over_molecules
-   USE neb_methods,                     ONLY: neb
-   USE negf_methods,                    ONLY: do_negf
-   USE offload_api,                     ONLY: offload_get_chosen_device,&
-                                              offload_get_device_count
-   USE optimize_basis,                  ONLY: run_optimize_basis
-   USE optimize_input,                  ONLY: run_optimize_input
-   USE pint_methods,                    ONLY: do_pint_run
-   USE pw_fpga,                         ONLY: pw_fpga_finalize,&
-                                              pw_fpga_init
-   USE pw_gpu,                          ONLY: pw_gpu_finalize,&
-                                              pw_gpu_init
-   USE qs_environment_types,            ONLY: get_qs_env
-   USE qs_linres_module,                ONLY: linres_calculation
-   USE qs_tddfpt_module,                ONLY: tddfpt_calculation
-   USE reference_manager,               ONLY: print_all_references,&
-                                              print_format_html
-   USE rt_propagation,                  ONLY: rt_prop_setup
-   USE sirius_interface,                ONLY: cp_sirius_finalize,&
-                                              cp_sirius_init
-   USE swarm,                           ONLY: run_swarm
-   USE tamc_run,                        ONLY: qs_tamc
-   USE tmc_setup,                       ONLY: do_analyze_files,&
-                                              do_tmc
-   USE vibrational_analysis,            ONLY: vb_anal
+   USE atom, ONLY: atom_code
```

Comment:
Same here, is the indentation change intentional?

Reply:
Won't fix. Let's deal with this on CP2K's side.